### PR TITLE
Add test for querying all images in the database -- check total number

### PR DESCRIPTION
**Pull request by frank-24**
test `GET`: querying all images in the database.
1. test case: check if the expected total number of items is correct (see "total" inside "_meta" of returned response)

### CODE REVIEW

File: test/testGetPost.py [hunk 1]
```diff
@@ -18,10 +17,42 @@ def getRequest(postResponse, url):
...
+        # 1. initialize param. change to others for later use
+        url = "http://localhost:80/scenarios"
+        log= logging.getLogger( "SomeTest.testSomething" )
+
+        count = 0
+
+        self.assertTrue( count == 0 )
+
+        for fileName in glob(pattern):
+            count = count + 1
```
Comment (chrisgorgo) [inline test/testGetPost.py]: `len(glob(pattern))`

File: test/testGetPost.py [hunk 2]
```diff
@@ -18,10 +17,42 @@ def getRequest(postResponse, url):
...
+
+    def test_01_GETAllData(self):
+        # 1. initialize param. change to others for later use
+        url = "http://localhost:80/scenarios"
+        log= logging.getLogger( "SomeTest.testSomething" )
+
+        inputCount = 0
+        for fileName in glob(pattern):
+            with open(fileName) as fp:
+                inputCount = inputCount + 1
```
Comment (chrisgorgo) [inline test/testGetPost.py]: `inputCount += 1`

**Review by chrisgorgo — CHANGES_REQUESTED** [review]
minor corrections - otherwise looks good!

**Comment by frank-24** [comment]
Thanks! I've updated.

**Pull request merged**